packages/grafana-data/src/field/fieldDisplay.test.ts

```diff
@@ -102,7 +102,7 @@ describe('FieldDisplay', () => {
   it('Should return field with default text when no mapping or data available', () => {
     const options = createEmptyDisplayOptions();
     const display = getFieldDisplayValues(options);
-    expect(display[0].display.text).toEqual('No data');
+    expect(display[0].display.text).toEqual('');
     expect(display[0].display.numeric).toEqual(0);
```

Comment:
i think for your scenario you need to also add field config with displayName set to empty string

Reply:
I have added new test case 'Should return empty when no field mapped and no data'